### PR TITLE
Changed maxwear to uses in lua_api.txt

### PR DESCRIPTION
**Pull request by DonBatman**
#4444

### CODE REVIEW

File: doc/lua_api.txt [hunk 1]
```diff
@@ -3489,8 +3489,7 @@ Definition tables
             max_drop_level=0,
             groupcaps={
                 -- For example:
-                snappy={times={[2]=0.80, [3]=0.40}, maxwear=0.05, maxlevel=1},
-                choppy={times={[3]=0.90}, maxwear=0.05, maxlevel=0}
+				choppy={times={[1]=2.50, [2]=1.40, [3]=1.00}, uses=20, maxlevel=2},
```
Comment (paramat) [inline doc/lua_api.txt]: Indentation seems weird, it should be done with spaces not tabs here.

**Comment by paramat** [comment]
See line comments, otherwise +1.

**Comment by DonBatman** [comment]
@paramat - Fixed

**Comment by paramat** [comment]
Spaces around '=' too please.

File: doc/lua_api.txt [hunk 2]
```diff
@@ -3472,26 +3472,26 @@ Definition tables
...
-                            {bendy=2, snappy=1},
-                            {hard=1, metal=1, spikes=1}
+                        e.g. {wool = 1, fluffy = 3}
+                            {soil = 2, outerspace = 1, crumbly = 1}
+                            {bendy = 2, snappy = 1},
+                            {hard = 1, metal = 1, spikes = 1}
         inventory_image = "default_tool_steelaxe.png",
         wield_image = "",
-        wield_scale = {x=1,y=1,z=1},
+        wield_scale = {x = 1,y = 1,z = 1},
```
Comment (paramat) [inline doc/lua_api.txt]: Spaces after commas.

**Comment by paramat** [comment]
Line comment, otherwise good.

**Comment by paramat** [comment]
Sorry just one more thing at L3490:
```groupcaps={```
Then i'll merge this before conflicts arise.

**Comment by DonBatman** [comment]
Sorry for missing that one.

**Comment by paramat** [comment]
:+1: No problem.

**Comment by paramat** [comment]
986d70ccecb09fcbd247758643a4d623c9c4e1b8

**Pull request closed without merging**